### PR TITLE
Adding Instructions for Contributing to Servo Using The Janitor

### LINKED ISSUE #12879: Add Instructions for Contributing to Servo Using The Janitor
I have been using The Janitor (https://janitor.technology) for some time now. It's evident it would greatly simplify the process when creating the environment to new comers to Servo.

I think it would be great to have a link to [https://janitor.technology](url) in the ReadMe.md file.

Tags to add please: E-Easy, A-Documentation, C-Assigned

### PR DESCRIPTION
**Pull request by Coder206**
---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [x] These changes fix #12879 (github issue number if applicable).

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/12882)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -223,4 +226,4 @@ There are lots of mach commands you can use. You can list them with `./mach
 --help`.
 
 
-The generated documentation can be found on http://doc.servo.org/servo/index.html
+The generated documentation can be found on http://doc.servo.org/servo/index.html
```
Comment (KiChjang) [inline README.md]: I have zero idea what changed on this line.

Comment (emilio) [inline README.md]: The trailing newline.

**Comment by emilio** [comment]
Hi @Coder206, this looks good to me, but I wonder how much priority should we give to `janitor.technology` instead of local development.

Personally, I'd move the advice a bit down, in another section maybe. That being said, I'd like to defer to @larsbergstrom or @jdm about where that text should live.

Thanks for the PR!

**Comment by Coder206** [comment]
@emilio Thanks! My pleasure! 😀 I understand what you mean. To me it makes more sense when read with the changes from #12880

**Comment by emilio** [comment]
@Coder206: I see, sorry, I missed that! Maybe you could merge the two PRs together? Otherwise one is going to break the other. Also, that way it's easier to see how the final README would look like :-)

**Comment by Coder206** [comment]
@emilio It's fine! The reason the two are separate is because, the changes at #12880 are more oriented towards linking the OSs and organizing the page better. Whereas, this PR is strictly for The Janitor's addition. In the case of code breaking perhaps it would be best to get some feedback on #12880 before continuing as it dictates a little how The Janitor's content will be written.

File: README.md (hunk 1 above)
Comment (nox) [inline README.md]: Yeah, please revert that change.

**Comment by Coder206** [comment]
@nox What is the code change required?

**Comment by nox** [comment]
The missing newline at end of file, cf. my comment earlier.

**Comment by Coder206** [comment]
@nox Sorry I didn't see that!

**Comment by KiChjang** [comment]
@Coder206 Side note - you don't have to reply every time on GitHub, all reviewers/people who subscribed to the Servo repository will get notifications/emails for every comment you make.

**Comment by Coder206** [comment]
@nox I copied the old readme file and added my changes. Is this what you were looking for?

**Comment by Coder206** [comment]
@emilio In about 7 hours, I hope to have finished with #12880 and then we can get a better understanding of where to place The Janitor's instructions.

**Comment by Coder206** [comment]
@jdm Please note this PR must be merged after #12880 is closed (merged).

**Comment by jdm** [comment]
I thought the Janitor was still in closed alpha? I don't see any point to adding this to the readme until it's actually a viable option.

**Comment by Coder206** [comment]
@jdm Yes, it's per invitation only for the moment. I understand your point. Should this PR be closed then reopened when The Janitor matures?

**Comment by jdm** [comment]
Yes, that makes sense to me.

**Pull request closed without merging**